cms/navigation/blocks.py

```diff
@@ -47,3 +47,15 @@ class ColumnBlock(StructBlock):
     class Meta:
         icon = "list-ul"
         label = _("Column")
+
+
+class LinksColumn(StructBlock):
+    title = CharBlock(required=True, help_text=_("Title for this column"))
```

Comment:
```suggestion
    title = CharBlock(required=True, label=_("Column title"))
```

aside: we have https://jira.ons.gov.uk/browse/CMS-387 which will remove the translation marker for strings coming from Python, unless they are for the front-end

Reply:
[1067893](https://github.com/ONSdigital/dis-wagtail/pull/111/commits/1067893aa8ff7870288467daf7a4a8d5b3ba8869)